examples/tutorial/src/vote.ts

```diff
@@ -0,0 +1,20 @@
+import { Wallet } from '@ethersproject/wallet';
+import { VocdoniSDKClient, Vote } from '@vocdoni/sdk';
+
+export async function castVotes (voters: Wallet[], client: VocdoniSDKClient) {
```

Comment:
This code works because it's casting votes one by one and therefore using the same client object. Isn't here a reason why we couldn't use a `Promise.all`? Maybe I would do it so, declaring a new client for each voter.

Reply:
Ok, thanks for the tip.